lib/sli/rcsinfo.sli

```diff
@@ -0,0 +1 @@
+statusdict /rcsinfo (no_rcsinfo_available) put
```

Comment:
I think this file should not be committed, should it? Isn't it automatically generated?